configuration/configuration_test.go

```diff
@@ -102,6 +107,36 @@ func TestBlobCorrectPayload(t *testing.T) {
 	testBlob(t, &payload, []byte(fullValidConf), true)
 }
 
+//equalNetSlice creates copies of slices received
+// sort them and then check elements are equal
+func equalNetSlice(slice1, slice2 []string) bool {
```

Comment:
@mrkz So you want to compare if the 2 subnets slices are equal, right ?
2 options here:

1. The order of the subnets matters, i.e `[a, b] != [b, a]` for us. In which case you can simply call `reflect.DeepEqual(slice1, slice2)`

2. The order of the subnets does not matter, i.e. `[a, b]` and `[b, a]` are equivalent for us. In that case you can sort and copy the 2 slices and then call  `reflect.DeepEqual(sortedSlice1, sortedSlice2)`.

@mcastelino When building the mgmt and compute list of subnets for `NetworkConfig`, does the order of the subnets matter ?